### ISSUE TITLE
BUG: renaming with `onyo mv` prints message calling it "moved"

### ISSUE THREAD

**Issue by TobiasKadelka** [psyinfra/onyo#418]
```
❱ onyo mv accounting account
The following will be moved:
/Users/tkadelka/none-existing-folder/accounting -> account
```

The call does rename a folder, but the message "will be moved"

**Comment by aqw** [comment]
I have tested this: the commit message subject, contents, and operations log are all correct. It's just the user-facing message in the terminal that's incorrect.

**Comment by TobiasKadelka** [comment]
I solved this a while ago and forgot to close the issue; here the same command that lead to opening the issue:
```
tkadelka in ~/onyo on git:master
❱ onyo mv accounting account
The following will be renamed:
/Users/tkadelka/onyo/accounting -> /Users/tkadelka/onyo/account
Save changes? No discards all changes. (y/n) 
```

**Comment by bpoldrack** [comment]
AFAICT addressed by PR #479
